src/browser/CMakeLists.txt

```diff
@@ -16,9 +16,10 @@
 
 if(WITH_XC_BROWSER)
     include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
-    find_package(sodium 1.0.12 REQUIRED)
+    include_directories(SYSTEM ${sodium_INCLUDE_DIR})
```

Comment:
Shouldn't these be ```target_include_directories``` ??

Reply:
I don't think so. We want to use libsodium in all of KeePassXC, not only the main target.

Reply:
Then this should be in the main CMakeLists.txt not conditionally added

Reply:
Yeah, now that you're saying it, I don't know why I didn't move it out of the if condition.

Reply:
Alright, I did add it to the global CMakeLists.txt. This one's just redundant and I removed it.